logos/src/logos/main.py

```
@@ -16,7 +16,7 @@
 from fastapi.middleware.cors import CORSMiddleware
 from fastapi.responses import JSONResponse
 from logos.auth import authenticate_logos_key
-from logos.role_auth import require_logos_admin_key, require_app_admin_or_above
+from logos.role_auth import require_logos_admin_key, require_app_admin_or_above, require_logos_admin
```

Flake8 (7.3.0)

[error] 'logos.role_auth.require_logos_admin' imported but unused

(F401)